deployments/datahub/images/default/environment.yml

```diff
@@ -250,51 +186,13 @@ dependencies:
   # https://github.com/berkeley-dsep-infra/datahub/issues/1981
   - ipycanvas==0.9.0
 
-  # data100 scientific packages
-  - ray==2.8.0
-  - xlrd==2.0.1
-
-  # data100 visualization
-  - colorlover==0.3.0
-  - cufflinks==0.17.3
-  - altair==4.1.0
-
-  # data100 tools to access things
-  - tweepy==3.10.0
-  - pytz==2021.1
-  - psycopg2==2.9.1
-
-  # data100 teaching
-  - jassign==0.0.7
-  - dsassign==0.0.8
-
-  # data102
-  - dask==2021.10.0
-  - distributed==2021.10.0
-  - keras-applications==1.0.8
-  - keras-preprocessing==1.1.2
-  - keras==2.11.0
-  - keras-vis==0.4.1
-  - plotly-express==0.4.1
-  - cytoolz==0.11.0
-  - pyro-ppl==1.7.0
-  - lime==0.2.0.1
-  - shap==0.39.0
-
   # prob140 2021 Spring
```

Comment:
We should either delete prob140 dependencies here, or preserve them, delete the prob140 hub, and have the prob140 instructor use datahub. The prob140 hub was created in order to isolate GSI access to student materials -- i.e. when data8 GSIs might be prob140 students, but this is now addressed by scopes and roles. Deleting the prob140 hub would require @balajialg / @ericvd-ucb to work with its course staff to ease the migration.

Deleting the dependencies here and preserving prob140 hub (or just doing nothing at this point) is the easier option, but I don't think its the right one long term.

Reply:
yep, you're right and we should eventually sunset the prob140 hub.  i'll comment these out and put a note in the comment.  i also just created https://jira-secure.berkeley.edu/browse/DH-203 to internally track this.